contents/handbook/engineering/how-to-qa.md

```diff
@@ -6,70 +6,74 @@ showTitle: true
 
 ## Why QA something?
 
-We’ve found that almost every time we QA something, we find something that would have been confusing/annoying to our users. Hence it’s worth doing QA for everything. There are very few excpetions
+We’ve found that almost every time we do Quality Assurance on something, we find something that would have been confusing/annoying to our users. Hence it’s worth doing QA for everything. 
+
+There are very few excpetions to this rule.
 
 ## What does good QA look like?
 - You understand what has changed
 - You understand why that change was made, so you have context
 - You understand which parts of the system could be affected
-    - For example, if a change is made to the way a button on the login form works, that might also appear in the /setup page, so you should check what happens/.
-- You should have a high standard -- but be flexible
-    - We want to build the best possible product. In terms of priority (most important first), the feature
-        - works;
-        - is obvious how it works;
-        - doesn't have weird glitches or bugs;
-        - doesn’t cause bugs elsewhere;
+    - For example, if a change is made to the way a button on the login form works, that might also appear in the /setup page, so you should check what happens.
+- You should have a high standard -- but be flexible.
+    - We want to build the best possible product. In terms of priority (most important first), the following should be true about a new feature:
+        - It works.
+        - It is obvious how it works.
+        - It doesn't have weird glitches or bugs.
+        - It doesn’t cause bugs elsewhere.
+
 
+## Setting up Heroku Test Environment 
 
-## How to do QA
+**Note: This is now done automatically.**
```

Comment:
The process below _is_ the automatic process so this is slightly confusing. All it's saying is "if you want to QA this on a Heroku test environment, go to the PR and click on the env"

![image](https://user-images.githubusercontent.com/1727427/89798028-ee852000-db2b-11ea-96f7-73d5cfb74352.png)

Reply:
Hmm yeah... So what's the suggestion regarding this section?